frontend/src/app/schedule/scheduled-day/scheduled-day.component.html

```diff
@@ -3,7 +3,7 @@
     *ngIf="item.classroomId === classroom"
     style="backgroundColor: {{ item.groupColor }}"
   >
-    <div class="popup" (click)="showNote()" (mouseover)="pauseTimeout()" (mouseleave)="startTimer()">
+    <div class="popup z-20" (click)="showNote()" (mouseover)="pauseTimeout()" (mouseleave)="startTimer()">
```

Comment:
i assume this is to prevent the note from appearing behind the top date/classroom row, but this also causes the text of the scheduled days to appear above the schedule control buttons and navbar when the schedule gets scrolled